include/swift/DWARFImporter/DWARFImporter.h

```diff
@@ -27,6 +29,25 @@ namespace clang {
 
 namespace swift {
 
+/// This interface is implemented by LLDB to serve as a fallback when Clang
+/// modules can't be imported from source in the debugger.
+///
+/// During compile time, ClangImporter-imported Clang modules are compiled with
+/// -gmodules, which emits a DWARF rendition of all types defined in the module
+/// into the .pcm file. On Darwin, these types can be collected by
+/// dsymutil. This delegate allows DWARFImporter to ask LLDB to look up a Clang
+/// type by name, synthesize a Clang AST from it. DWARFImporter then hands this
+/// Clang AST to ClangImporter to import the type into Swift.
+class DWARFImporterDelegate {
+public:
+  virtual ~DWARFImporterDelegate() {}
```

Comment:
Nitpick: `= default` allows a tiny bit more optimization in some cases.

Reply:
https://github.com/apple/swift/pull/26637